### ISSUE TITLE
warning about ofColor constants.

### ISSUE THREAD

**Issue by 2bbb** [openframeworks/openFrameworks#5546]
maybe after i did update Xcode 8.3.1, i have been got some warning like.

```
/Users/2bit/Program/oFs/of_v0.9.8_osx_release/libs/openFrameworks/graphics/ofGraphics.h:643:109: Instantiation of variable 'ofColor_<unsigned char>::black' required here, but no definition is available
```

i added the line `extern template struct ofColor_<unsigned char>;` on top of `ofGraphics.h` (with reference to [this](https://github.com/fmtlib/fmt/issues/318).)
then warning was go out.

but I don't have confidence...

does anyone know what this is correct or not, or other better solution?

**Comment by bakercp** [comment]
I can confirm these warnings, but I haven't had a chance to research the correct solution yet.

**Comment by mikechambers** [comment]
I am still seeing this issue in .9.8 on OSX

**Comment by arturoc** [comment]
yes this was fixed in github post 0.9.8 so the fix is in the nightlies and will be in 0.10 but 0.9.8 still has this issue
